_data/banking.yml

```diff
@@ -163,7 +163,7 @@ websites:
       phone: Yes
       doc: https://www.mybankofinternet.com/tob/live/usp-core/static/help.html#login_security
 
-    - name: Caisse D'Epargne
```

Comment:
Looks like you're not up to the with our master branch. I suggest you pull/revert before we merge this PR.